manifests/common/crd/odhdashboardconfigs.opendatahub.io.crd.yaml

```diff
@@ -140,8 +150,12 @@ spec:
                                 type: string
                               memory:
                                 type: string
-                # Hardware Profiles going GA will help remove these
+                  x-kubernetes-validations:
+                    - rule: self == oldSelf
```

Comment:
we should do this for all these 
```
self == oldSelf || self == null
```

this will allow it to be removed also

Reply:
It allows it to be removed this way, seems like validation runs only when the field is present, and removal is allowed cause those fields aren't set as required.
Also adding `self == null` throws an error